sadcowpng/sharedspaces/models.py

```diff
@@ -1,3 +1,52 @@
 from django.db import models
+from .forms import Noise_Level_Choices
 
-# Create your models here.
+
+# All information that will be sent to space
+class Space(models.Model):
+    space_name = models.CharField(max_length=500)
+    space_description = models.CharField(max_length=1000)
+    space_max_capacity = models.IntegerField()
+    space_noise_level_allowed = models.IntegerField()
+    space_noise_level = models.IntegerField()
+    space_wifi = models.BooleanField()
+    space_restrooms = models.BooleanField()
+    space_food_drink = models.BooleanField()
+
+    # space_open = models.BooleanField()
```

Comment:
Same as earlier comment, we don't want to merge commented code. All code should serve a purpose

Reply:
Will keep it for later development. will remove in future tho